spanner/google/cloud/spanner/__init__.py

```diff
@@ -18,6 +18,13 @@
 import pkg_resources
 __version__ = pkg_resources.get_distribution('google-cloud-spanner').version
 
+from google.cloud.spanner._helpers import BOOL_PARAM_TYPE
```

Comment:
I feel weird about plucking members out of a private module to expose. Do we have a `spanner.types` module?